assets/scss/_base.scss

```diff
@@ -687,13 +687,56 @@ section#cncf {
 }
 
 // Features
-#features {
+body.td-home section.features-container {
   padding-top: 140px;
+
   background-color: $light-grey;
   background-image: url(/images/wheel.svg);
   background-position: center 60px;
   background-repeat: no-repeat;
   background-size: 60px;
+
+  .k8s-features-heading {
+    text-align: center;
+    color: $blue;
+  }
+
+  & > div {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+    align-content: center;
+    gap: 1.8em;
+
+    margin-top: 1.8em;
+
+    max-width: clamp(75em, 25cm, 90vw);
+    margin-left: auto;
+    margin-right: auto;
+
+    background: initial;
+
+    & > .feature-box {
+        h4 {
+          line-height: normal;
+          margin-bottom: 1.9em;
+        }
```

Comment:
Looks like these lines of CSS are targeting the same element as the later lines in the file. Are they redundant?

https://github.com/kubernetes/website/blob/8924f40f2d004a25adfbd0bc0496a45648a263d0/assets/scss/_base.scss#L708-L711

Reply:
I dropped them. Thanks.